tensorly/tests/test_backend.py

```diff
@@ -327,6 +403,13 @@ def test_clips_all_negative_tensor_correctly():
     assert tl.all(clipped_tensor == 0)
 
 
+def test_clips_all_negative_tensor_correctly_paddle():
```

Comment:
Could you please add a short docstring describing this test? Why does it have paddle in the name if it addresses a pytorch regression?

Reply:
Sorry for that I just copy and paste at that time, it is more suitable to add if-else for paddle in test_set_backend 👍